### PR TITLE
Add feature Remove-SqlBackupFromDisk

### PR DESCRIPTION
**Pull request by cjsommer**
New cmdlet to cleanup sql backup files from local disk. Functionality is similar to standard maintenenace plan cleanup tasks.

### CODE REVIEW

File: functions/Remove-SqlBackupFromDisk.ps1 [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+            Where-Object {$_.LastWriteTime -lt $RetentionDate}
+            
+        # Filter out unarchived files if -CheckArchiveBit parameter is used
+        if ($CheckArchiveBit.IsPresent) {
+            Write-Output 'Removing only archived files'
+            $FilesToDelete = $FilesToDelete | Where-Object {$_.attributes -notmatch "Archive"} 
+        }
+        
+        # Perform the deletion or show which file will be deleted if WhatIf is used
+        If ($Pscmdlet.ShouldProcess($env:computername, "Removing backup files from '$BackupFolder\*'")) {
```
Comment (ctrlbold) [inline functions/Remove-SqlBackupFromDisk.ps1]: To add more granularity, remove this shouldprocess, and go file by file
foreach ($file in $filestodelete)  { If ($Pscmdlet.ShouldProcess($env:computername, "Deleting $file from $backupfolder'")) {

File: functions/Remove-SqlBackupFromDisk.ps1 [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+	Param (
+		[parameter(Mandatory = $true)]
+        [ValidateScript({Test-Path $_ -PathType 'Container'})]
+		[string]$BackupFolder ,
+
+		[parameter(Mandatory = $true)]
+		[string]$BackupFileExtenstion ,
+
+		[parameter(Mandatory = $false)]
+		[string]$RetentionPeriod = '48h' ,
```
Comment (ctrlbold) [inline functions/Remove-SqlBackupFromDisk.ps1]: This seems very short. Can we make the default longer? 60d perhaps? Or poll the channel, but I think 30d should be minimum. Or should this be mandatory with no assumptions?

File: functions/Remove-SqlBackupFromDisk.ps1 [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+
+#>
+	[CmdletBinding(SupportsShouldProcess=$true)]
+	Param (
+		[parameter(Mandatory = $true)]
+        [ValidateScript({Test-Path $_ -PathType 'Container'})]
+		[string]$BackupFolder ,
+
+		[parameter(Mandatory = $true)]
+		[string]$BackupFileExtenstion ,
```
Comment (ctrlbold) [inline functions/Remove-SqlBackupFromDisk.ps1]: I think it'd be safe to give this a default of .bak. What about you?

Comment (cjsommer) [inline functions/Remove-SqlBackupFromDisk.ps1]: I think that would be a safe option. Either that or once again make them choose which makes them think about it.

**Comment by dansqldba** [comment]
Discussed with @cjsommer and this PR is not required, replaced by #237.

**Pull request closed without merging**